test_app/tests/lib/routers/test_association_resoure_router.py

```diff
@@ -186,12 +174,33 @@ def test_association_router_related_viewset_all_mapings(db):
     expected_urls = [
         'my_test_basename-detail',
         'my_test_basename-list',
-        'my_test_basename-teams-detail',
```

Comment:
I'm not going to lie, this kind of test assertion is a little bit upsetting. This URL should not be registered, but someone probably added it in just to make the test pass.